simul/build.go

```diff
@@ -299,8 +330,13 @@ func mkTestDir() {
 	}
 }
 
-func testFile(name string) string {
-	return "test_data/" + name + ".csv"
+func testFile(name string, index int) string {
```

Comment:
I find the name of this function misleading. Maybe `getTestFile(..)` would be more appropriate?

Reply:
Renamed as `generateResultFileName`